pkg/plugin/v3/scaffolds/webhook.go

```diff
@@ -78,13 +81,22 @@ func (s *webhookScaffolder) scaffold() error {
 You need to implement the conversion.Hub and conversion.Convertible interfaces for your CRD types.`)
 	}
 
-	if err := machinery.NewScaffold().Execute(
+	if err := machinery.NewScaffold(s.plugins...).Execute(
 		s.newUniverse(),
 		&api.Webhook{Defaulting: s.defaulting, Validating: s.validation},
 		&templates.MainUpdater{WireWebhook: true},
 	); err != nil {
 		return err
 	}
 
+	if s.defaulting || s.validation {
```

Comment:
why we scaffold the suite just in these both cases?

Reply:
The conversion scenario doesn't add anything in the webhook.go it creates. No default functions and markers around those are added for conversion. Hence it doesn't create any entry for conversion Kind into the validating and mutating configuration in the folder config/webhook.This the reason no need to add the suite changes for conversion webhook. You can check the conversion webhook here: `testdata/project-v3-multigroup/apis/ship/v1beta1/frigate_webhook.go`

Reply:
There is a pending PR #1664 around implementing conversion webhook properly. Even after that this suite should not go there. There is a pending PR in controller-runtime which adds support of conversion webhook in envtest. We can add the different suite for conversion webhook once this [PR is merged](https://github.com/kubernetes-sigs/controller-runtime/pull/999)

Reply:
so, could you please add a todo comment for we do not forget it?
Also, could you ping this pr in the #1664 for the author knows that if it gets merged this if required to be changed as it will be described in the todo comment?

Reply:
Done